tdvt/tdvt/exprtests/standard/setup.string.contains.txt

```diff
@@ -1,2 +1,3 @@
 CONTAINS(str2, "e")
 CONTAINS(str1, "IND")
+CONTAINS(str2, LEFT(str2, 0))
```

Comment:
I recommend adding this as a new setup file, something like setup.string.contains.empty.txt. That way you could add two new expected files and leave the originals unchanged. This simplifies maintenance since you don't have to duplicate existing test case results in order to add a new expected file.